components/Select/Select.stories.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 
-import { BaseSelect, SelectProps, SelectVariants } from './Select';
+import { Select, SelectProps, SelectVariants } from './Select';
```

Comment:
Stories improvements (a bit unrelated to this PR, feel free to postpone into a new issue):
- stories with `value` prop lack real world interactivity (mainly `Ghost`), could you replace `value` by `defaultValue` ?
- It could be nice to rename `Large` story into `Sizes` and either give a select control for sizes, or display all size variants, I was misled into thinking there's not story for small variant